app/controllers/api/v1x0/workflows_controller.rb

```diff
@@ -76,6 +73,16 @@ def rbac_scope(relation)
 
         relation
       end
+
+      def resource_object_params
+        params.slice(:object_type, :object_id, :app_name).to_unsafe_h
```

Comment:
@hsong-rh do you want to save this using

@resource_object_params ||= params.slice(:object_type, :object_id, :app_name).to_unsafe_h

Because in the for_resource_object? you this method is going to get called twice